packages/buffetjs-styles/src/components/Button/index.js

```diff
@@ -58,6 +58,7 @@ const Button = styled.button`
       padding: 0 ${sizes.button.padding.small};
       svg {
         fill: ${colors.blue};
+        vertical-align: initial;
```

Comment:
Are you sure that this won't mess with the Strapi `administration panel` ? I just want to make sure we won't have regressions again.